doc/api/next_api_changes/behavior/22229-TAC.rst

```diff
@@ -0,0 +1,18 @@
+AritistList proxies copy contents on iteration
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+When iterating over the contents of the the dynamically generated proxy lists
+for the Artist-type accessors (see :ref:`Behavioural API Changes 3.5 - Axes
+children combined`), a copy of the contents is made.  This ensure that artists
+can safely be added or removed from the Axes while iterating over their children.
+
+This is a departure from the expected behavior of mutable iterable data types
+in Python -- iterating over list while mutating it has surprising consequences
+and dictionaries will error if they change size during iteration.  In this case
+because all of the accessors are filtered views of the same underlying list, it
+is possible for seemingly unrelated changes, such as removing a Line while
+iterating over patches, we have opted to make a copy of the relevant children
+before yielding them to the user.
```

Comment:
It is possible for seemingly unrelated changes to what? There seems to be a clause missing.

Reply:
Is that any clearer?